### PR TITLE
Add draft of video content

### PR DESCRIPTION
**Pull request by leonardkong**
Add a first-cut of video content

### CODE REVIEW

File: Videos/Smart Configurator Videos_filtered.htm [hunk 1]
```diff
@@ -0,0 +1,150 @@
+<html xmlns:v="urn:schemas-microsoft-com:vml"
+xmlns:o="urn:schemas-microsoft-com:office:office"
+xmlns:w="urn:schemas-microsoft-com:office:word"
+xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
+xmlns="http://www.w3.org/TR/REC-html40">
+
+<head>
+<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
```
Comment (nam-ng) [inline Videos/Smart Configurator Videos_filtered.htm:8]: It is better to be "UTF8" for charset

File: Videos/Smart Configurator Videos_filtered.htm [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+<meta name=ProgId content=Word.Document>
+<meta name=Generator content="Microsoft Word 15">
```
Comment (nam-ng) [inline Videos/Smart Configurator Videos_filtered.htm:10]: Should use "filtered" when exporting html from MS Word

**Review by nam-ng — COMMENTED** [review]
May need to re-export htm file from MS Word for better encoding and simpler content

File: Videos/Smart Configurator Videos_filtered.htm (hunk 2 above)
Comment (leonardkong) [reply Videos/Smart Configurator Videos_filtered.htm]: Done

File: Videos/Smart Configurator Videos_filtered.htm (hunk 1 above)
Comment (leonardkong) [reply Videos/Smart Configurator Videos_filtered.htm]: Done

**Pull request merged**